services-gateway-client-transport/src/main/java/io/scalecube/services/gateway/transport/KeepaliveMessage.java

```diff
@@ -0,0 +1,10 @@
+package io.scalecube.services.gateway.transport;
+
+import io.scalecube.services.api.ServiceMessage;
+
+public class KeepaliveMessage {
```

Comment:
`KeepaliveMessage` is redundant.

[Websocket specification (RFC 6455)](https://tools.ietf.org/html/rfc6455#section-5.5.2) defines that endpoint MUST send a `Pong` frame as a response to incoming `Ping` frame.

Both `netty` and `reactor-netty` supports this feature out of the box:
- netty 
https://github.com/netty/netty/blob/43252a61352f2785c19e321c67d5ac87aee4d172/codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketProtocolHandler.java#L48-L53
- reactor-netty
https://github.com/reactor/reactor-netty/blob/cadd13c11bb01d946164d61edb09a4bd472e0184/src/main/java/reactor/netty/http/server/WebsocketServerOperations.java#L147-L151

So, looks like it is enough just to send existing [PingWebsocketFrame](https://github.com/netty/netty/blob/4.1/codec-http/src/main/java/io/netty/handler/codec/http/websocketx/PingWebSocketFrame.java).